Features chapter 7

IIKUYY opened the pull request:
## what

- Agregar el resumen del capitulo 7 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

IIKUYY committed:
Feat: Actualizar TOC de navgación

## what

- Agregar vínculos a los menús

## why

- Mantener el uso correcto de TOC de navegación

## references
-

IIKUYY committed:
Docs: Agregar resumen del capitulo 7

## what

- Agregar el resumen del capitulo 7 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

semanticmx commented on Ch7/Ejemplo/index.html:
```diff
@@ -0,0 +1,27 @@
+      </div>
+    </center>
+  </body>
+  <script src="main.js"></script>
```
esta perfecto solo que todos los tags deben ir dentro del body.

en lugar de:

```html
</body>
   <script src="main.js"></script>
```

debe ser

```html
    <script src="main.js"></script>
</body>
```

semanticmx commented on Ch7/Ejemplo/main.js:
```diff
@@ -0,0 +1,17 @@
+var counterVal = parseInt(localStorage.getItem('counterValue')) || 0;
```
por convensión vamos a listar primero las funciones y después ya las acciones por ejemplo, la definición de las funciones iría hasta arriba y ya la parte de la creación de counterVal y la llamada a updateDisplay quedaría hasta abajo.

semanticmx reviewed (changes requested):
muy bien, igual trata de agregar los screenshot al README.md con la implementación del html/JS/CSS que vayas haciendo en los capítulos

IIKUYY committed:
Chores: Realizar las tareas dadas

## what

- Mover las funciones al principio del código
- Mover el <script> adentro de la etiqueta body

## why

- Arreglar la semántica de los archivos
- Mejorar la optimización del código

## references
-

IIKUYY commented:
Cambios del código realizados, ¿las screenshots las pide en README.md general o en de cada capitulo?

semanticmx commented:
> Cambios del código realizados, ¿las screenshots las pide en README.md general o en de cada capitulo?

Yo diría que quedan bien en cada capítulo.

IIKUYY commented:
Vale, entonces las subo en este commit

IIKUYY committed:
Chores: Agregar Imagenes

## what

- Agregar Screenshots de los códigos en los capítulos donde faltaban

## why

- Mejorar la visibilidad del repositorio

## references
-

IIKUYY commented:
Imágenes agregadas
